frontend/src/scenes/hog-functions/configuration/HogFunctionTest.tsx

```diff
@@ -207,6 +207,9 @@ export function HogFunctionTest({ configurable }: { configurable: boolean }): JS
                             type="secondary"
                             onClick={() => {
                                 toggleExpanded()
+                                if (!(sampleGlobalsLoading && !fetchCancelled)) {
+                                    loadSampleGlobals()
```

Comment:
We had this before and it was a bad experience. In production this can take a long time to load and is super janky.

Can we instead move the button to be more prominent and call it "fetch real event" or something?

Reply:
![2025-05-23 at 12 17 36](https://github.com/user-attachments/assets/cc76abd4-3571-4110-9150-0a7897e18010)

Added a banner indicating it's an example event with a CTA to fetch a new event